### DISCUSSION TITLE
@blueprintjs/table evented button in a cell

### DISCUSSION THREAD

**Discussion by fredfortier** [palantir/blueprint#6422] [Q&A]
I want to put a button in each cell that triggers an event handler in the main table with a reference to the cell value. Based on the documentation, I'm thinking `EditableCell` with a custom cellRenderer that show a button instead of the underlying value. Clicking the button changes its mode to "loading" and triggers the `onChange` event of the table.

Is this the right way to think about this? Or is there an easier way to bubble custom events inside cells to the main table?

**Comment by Arkessler** [comment]
@fredfortier How did this wind up working out for you? I'm running into a very similar situation and so far the flow you've described seems like the only way to bubble events like this.
